main.py

```diff
@@ -1,10 +1,38 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players:
+        players = json.load(players)
+
+    for p_name in players:
```

Comment:
It would be better to use more descriptive variable name here
```suggestion
    for player_name in players:
```